test/ios_application_swift_test.sh

```diff
@@ -194,4 +194,82 @@ EOF
   fi
 }
 
+# Tests that swift_library builds with include_clang_rt and asan linker option
```

Comment:
Is there any information why the two tests above are disabled? Is `b/73547309` referring to some internal bug report?
The test work fine locally. If this issue is resolved, I can enable them, add a new test covering UBSAN, and add the tests for macOS as well.

Reply:
hrm yea i would just try to enable them at this point, idk either since it's an internal google one but it's highly likely that it works now